test/controllers/companies_controller_test.rb

```diff
@@ -114,4 +114,10 @@ def valid_company_params
     get :index
     assert_response 302
   end
+
+  test "should have the default company when accesing the companies page" do
```

Comment:
@godfreym you don't want to test what the default_company is, but rather that it's not nil. Think of a scenario where we decide to change the default company. this test will fail.